diff -aurBb /home/httpd/htdocs/users/kirichenko/cpp/var/ait_patch_orig/compare95113116849344e2eb9465f9c0 /home/httpd/htdocs/users/kirichenko/cpp/var/ait_patch_modif/compare95113116849344e2eb9465f9c0
--- /home/httpd/htdocs/users/kirichenko/cpp/var/ait_patch_orig/compare95113116849344e2eb9465f9c0	2011-07-26 15:55:34 +0300
+++ /home/httpd/htdocs/users/kirichenko/cpp/var/ait_patch_modif/compare95113116849344e2eb9465f9c0	2011-07-26 15:55:34 +0300
@@ -144,6 +144,10 @@
                 template = OptionTemplateFile;
                 group = 'file';
                 break;
+            case 'aitcustomer_image':
+                template = OptionTemplateCg;
+                group = 'cgfile';
+                break;                
             case 'drop_down':
             case 'radio':
             case 'checkbox':
@@ -225,6 +229,10 @@
                 template = OptionTemplateFile;
                 group = 'file';
                 break;
+            case 'aitcustomer_image':
+                template = OptionTemplateCg;
+                group = 'cgfile';
+                break;
             case 'drop_down':
             case 'radio':
             case 'checkbox':
@@ -266,6 +274,9 @@
                 });
                 selectOptionType.bindAddButton();
                 break;
+            case 'aitcustomer_image':
+                aitcgImageSelector.setOptionId(data.id, data.image_template_id);
+                break;
         }
 
         if (data.price_type) {
